wagtail/users/views/bulk_actions/assign_role.py

```diff
@@ -44,7 +44,7 @@ def prepare_action(self, objects):
 
     @classmethod
     def execute_action(cls, objects, **kwargs):
-        cls.role = role = kwargs.get('role', None)
+        role = kwargs.get('role', None)
```

Comment:
I feel like it's clearer to have these explicitly spelled out in the kwargs if they're a key part of the action